sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultVolumeSpec.java

```diff
@@ -18,6 +22,11 @@
     public static final String RESOURCE_NAME = "disk";
 
     private final Type type;
+
+    @NotEmpty
+    @NotBlank
+    @NotNull
```

Comment:
`@NotNull` is not needed.  both `@NotEmpty` and `@NotBlank` already enforce that.